### PR TITLE
Feature/496 change the theme

### LINKED ISSUE #496: Change the theme
- [ ] Change the colors 
- [ ] Change the typography (ask Jutta for details) :
https://www.figma.com/file/zadQiUwhMXr9632TBORG2P/Design-System---RN-App?node-id=363%3A1448

### PR DESCRIPTION
**Pull request by ahoseinian**
fixes #496 

Adds material design, fonts & colors

https://user-images.githubusercontent.com/1168983/140891200-df350c3b-d097-4c7d-84f9-bf4720a4f6ba.mp4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@eva-design/eva": "^2.1.1",
```
Comment (silva-fj) [inline package.json]: Do we need to keep this one?

**Review by silva-fj — DISMISSED** [review]
Does it make a significant difference in the UI? Can you add some screenshots to see the differences?

File: package.json (hunk 1 above)
Comment (ahoseinian) [reply package.json]: no, removed thanks

**Review by ishrak-imam — APPROVED** [review]
LGTM

**Review by silva-fj — APPROVED** [review]

**Pull request merged**